messaging/connectors/jms/src/main/java/io/helidon/messaging/connectors/jms/JmsConnector.java

```diff
@@ -122,6 +122,34 @@
     private final ExecutorService executor;
     private final Map<String, SessionMetadata> sessionRegister = new HashMap<>();
 
+    /**
+     * Creates a new instance of JmsConnector with the required configuration.
+     *
+     * @param config Helidon {@link io.helidon.config.Config config}
+     * @return the new instance
+     */
+    public static JmsConnector create(io.helidon.config.Config config) {
```

Comment:
This class should use a builder. If we need a separate constructor for CDI, so be it, but there should be a constructor with a builder, that allows explicit configuration of everything that can be configured through config.
In SE, there is always a choice for developer - either build everything by hand, or use a config, or a combination

For example the executor services must be configurable, as user may want to reuse an executor service for multiple components

Reply:
Great idea, fixed